src/Object/Api/Mastodon/Status.php

```diff
@@ -95,6 +96,8 @@ class Status extends BaseDataTransferObject
 	protected $card = null;
 	/** @var Poll|null */
 	protected $poll = null;
+	/** @var FriendicaExtension|null */
+	protected $friendica = null;
```

Comment:
```suggestion
	/** @var FriendicaExtension */
	protected $friendica;
```

You set it during the `__construct()` call. There is no `null` value possible.